check_mk_telegram-notify.sh

```diff
@@ -58,9 +58,30 @@ else
         echo "Invalid privacy parameter, check your Check_MK settings." >&2
 fi
 
+# Set an appropriate emoji for the current state
```

Comment:
I changed the UTF-8 encoded emojis in the source code to the "bytes (UTF-8)" notation. This makes more sense from my point of view for compatibility and customizing. The choice of emojis is a matter of taste.

# Set an appropriate emoji for the current state
# Feel free to change the emoji to your own taste. This is done by customizing the UTF8 code. Examples here: https://apps.timwhitlock.info/emoji/tables/u$
if [[ ${NOTIFY_WHAT} == "SERVICE" ]]; then
        STATE="${NOTIFY_SERVICESHORTSTATE}"
else
        STATE="${NOTIFY_HOSTSHORTSTATE}"
fi
case "${STATE}" in
    OK|UP)
        EMOJI=$'\xE2\x9C\x85' # white heavy check mark
        ;;
    WARN)
        EMOJI=$'\xE2\x9A\xA0' # warning sign
        ;;
    CRIT|DOWN)
        EMOJI=$'\xF0\x9F\x86\x98' # squared sos
        ;;
    UNKN)
        EMOJI=$'\xF0\x9F\x94\x84' # anticlockwise downwards and upwards open circle arrows
        ;;
esac